### ISSUE TITLE
KMZ can't be opened

### ISSUE THREAD

**Issue by simgislab** [nextgis-borsch/lib_gdal#24]
```
C:\Work>gdalinfo --version
GDAL 2.3.0, released 2018/05/04

C:\Work>gdalin
[test.zip](https://github.com/nextgis-borsch/lib_gdal/files/2030417/test.zip)
fo test.kmz
ERROR 4: `test.kmz' not recognized as a supported file format.
gdalinfo failed - unable to open 'test.kmz'.
```
Example [test.zip](https://github.com/nextgis-borsch/lib_gdal/files/2030420/test.zip)

**Comment by BishopGIS** [comment]
KMZ driver is not build now

**Comment by BishopGIS** [comment]
Fixed per a9d53afd50557eb697d0725a07f7d2c5e25ae836
